addons/bus/models/bus_presence.py

```diff
@@ -110,7 +110,7 @@ def _send_presence(self, im_status=None, bus_target=None):
                 self.env["bus.bus"]._sendone(
                     target,
                     "bus.bus/im_status_updated",
-                    {"im_status": im_status or presence.status, **identity_data},
+                    {"im_status": im_status or presence.user_id.forced_im_status or presence.status, **identity_data},
```

Comment:
Dito for the `forced_im_status` field. We might want to move the presence to mail as a prep.

Reply:
That sounds right, `im_status` does not really belong in bus anyway